docs/sql-programming-guide.md

```diff
@@ -965,6 +965,8 @@ Configuration of Parquet can be done using the `setConf` method on `SparkSession
     `parquet.compression` is specified in the table-specific options/properties, the precedence would be
     `compression`, `parquet.compression`, `spark.sql.parquet.compression.codec`. Acceptable values include:
     none, uncompressed, snappy, gzip, lzo, brotli, lz4, zstd.
+    Note that `zstd` needs to install `ZStandardCodec` before Hadoop 2.9.0, `brotli` needs to install
+    `brotliCodec`.
```

Comment:
@HyukjinKwon How about adding a link? Users may not know where to download it.
```
`brotliCodec` -> [`brotli-codec`](https://github.com/rdblue/brotli-codec)
```

Reply:
If the link looks expected to be rather permanent, it's fine.

Reply:
It is more clear to say "`zstd` requires ZStandardCodec to be installed".